From 46a8a04386ebe4886a40d2d92ff077ada44918de Mon Sep 17 00:00:00 2001
From: KO Myung-Hun <komh@chollian.net>
Date: Fri, 26 Jan 2018 19:07:21 +0900
Subject: [PATCH 17/24] Revert "qt: display directx rendering options only when
 vout is set to directx"

This reverts commit e7d528fe5621875c4df6b272a51742dbb5278de8.
---
 modules/gui/qt/components/simple_preferences.cpp | 18 ------------------
 modules/gui/qt/components/simple_preferences.hpp |  1 -
 2 files changed, 19 deletions(-)

diff --git a/modules/gui/qt/components/simple_preferences.cpp b/modules/gui/qt/components/simple_preferences.cpp
index afe30f247d..ecfa1dbb02 100644
--- a/modules/gui/qt/components/simple_preferences.cpp
+++ b/modules/gui/qt/components/simple_preferences.cpp
@@ -360,10 +360,6 @@ SPrefsPanel::SPrefsPanel( intf_thread_t *_p_intf, QWidget *_parent,
             CONFIG_BOOL( "video-deco", windowDecorations );
             CONFIG_GENERIC( "vout", StringList, ui.voutLabel, outputModule );
 
-            CONNECT( ui.outputModule, currentIndexChanged( int ),
-                     this, updateVideoOptions( int ) );
-            optionWidgets["videoOutCoB"] = ui.outputModule;
-
             optionWidgets["fullscreenScreenB"] = ui.fullscreenScreenBox;
             ui.fullscreenScreenBox->addItem( qtr("Automatic"), -1 );
             int i_screenCount = 0;
@@ -386,7 +382,6 @@ SPrefsPanel::SPrefsPanel( intf_thread_t *_p_intf, QWidget *_parent,
             CONFIG_BOOL( "directx-overlay", overlay );
             CONFIG_BOOL( "directx-hw-yuv", hwYUVBox );
             CONNECT( ui.overlay, toggled( bool ), ui.hwYUVBox, setEnabled( bool ) );
-            optionWidgets["directxVideoB"] = ui.directXBox;
 #else
             ui.directXBox->setVisible( false );
 #endif
@@ -410,8 +405,6 @@ SPrefsPanel::SPrefsPanel( intf_thread_t *_p_intf, QWidget *_parent,
                             snapshotsSequentialNumbering );
             CONFIG_GENERIC( "snapshot-format", StringList, ui.arLabel,
                             snapshotsFormat );
-
-            updateVideoOptions( ui.outputModule->currentIndex() );
          END_SPREFS_CAT;
 
         /******************************
@@ -943,17 +936,6 @@ SPrefsPanel::SPrefsPanel( intf_thread_t *_p_intf, QWidget *_parent,
 #undef CONFIG_BOOL
 }
 
-void SPrefsPanel::updateVideoOptions( int number )
-{
-    QString value = qobject_cast<QComboBox *>(optionWidgets["videoOutCoB"])
-                                            ->itemData( number ).toString();
-#ifdef _WIN32
-    if( optionWidgets["directxVideoB"] ) {
-        optionWidgets["directxVideoB"]->setVisible( ( value == "directdraw" ) );
-    }
-#endif
-}
-
 
 void SPrefsPanel::updateAudioOptions( int number)
 {
diff --git a/modules/gui/qt/components/simple_preferences.hpp b/modules/gui/qt/components/simple_preferences.hpp
index 856b88ba97..6390b21efd 100644
--- a/modules/gui/qt/components/simple_preferences.hpp
+++ b/modules/gui/qt/components/simple_preferences.hpp
@@ -117,7 +117,6 @@ private:
 /* Display only the options for the selected audio output */
 private slots:
     void lastfm_Changed( int );
-    void updateVideoOptions( int );
     void updateAudioOptions( int );
     void updateAudioVolume( int );
     void langChanged( int );
-- 
2.13.3

